Makefile

```diff
@@ -24,7 +24,11 @@ firectl.sha256:
 	sha256sum firectl > firectl.sha256
 
 firectl: $(SRCFILES)
+ifneq ($(STATIC_BINARY),)
+	CGO_ENABLED=0 go build -installsuffix cgo -a -ldflags "-s"
```

Comment:
Actually, I'm not sure we want `-ldflags "-s"`. We're not stripping symbols from the dynamically linked build. We should be consistent, and probably not strip by default since it's easy to do after the fact if desired.

Reply:
Done